app/helpers/load_balancer_helper/textual_summary.rb

```diff
@@ -31,18 +31,18 @@ def textual_description
   end
 
   def textual_listeners
-    @record.load_balancer_listeners.map do |x|
+    @record.load_balancer_listeners&.map do |x|
       "Load Balancer"\
         " #{LoadBalancerHelper.display_protocol_port_range(x.load_balancer_protocol, x.load_balancer_port_range)}"\
         ", Instance"\
         " #{LoadBalancerHelper.display_protocol_port_range(x.instance_protocol, x.instance_port_range)}"
-    end.join(" | ") if @record.load_balancer_listeners
+    end.join(" | ")
```

Comment:
undefined method `join' for nil:NilClass  (and on line 45)

Reply:
Yep, fixed.